Source/WebCore/page/Quirks.cpp

```diff
@@ -1716,6 +1716,19 @@ bool Quirks::shouldDisableNavigatorStandaloneQuirk() const
     return false;
 }
 
+// booking.com https://webkit.org/b/269875
+// FIXME: booking.com https://webkit.org/b/269876 when outreach has been successful.
+bool Quirks::shouldSendLongerAcceptHeaderQuirk(StringView host, LocalFrame* frame)
+{
+    if (frame && !frame->settings().needsSiteSpecificQuirks())
+        return false;
+
+    if (equalLettersIgnoringASCIICase(host, "booking.com"_s) || host.endsWithIgnoringASCIICase(".booking.com"_s))
```

Comment:
We don't usually need to ignore case for host checks. The URL class makes sure the host is lowercase already. For this reason also, it'd be nicer if the function took in a URL instead of a StringView.